subprojects/core/src/main/java/org/gradle/execution/taskgraph/DefaultTaskExecutionGraph.java

```diff
@@ -145,16 +150,16 @@ public void execute(Collection<? super Throwable> failures) {
             buildOperationExecutor.run(new NotifyTaskGraphWhenReady(this, graphListeners.getSource(), gradleInternal));
             hasFiredWhenReady = true;
         } else if (!graphListeners.isEmpty()) {
-            LOGGER.warn("Ignoring listeners of task graph ready event, as this build (" + gradleInternal.getIdentityPath() + ") has already run tasks.");
+            LOGGER.warn("Ignoring listeners of task graph ready event, as this build (" + gradleInternal.getIdentityPath() + ") has already executed work.");
```

Comment:
Is this the `task graph ready event` or the `graph ready event`?

Reply:
Still the task graph.